app/views/devise/registrations/new.html.erb

```diff
@@ -3,6 +3,16 @@
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
 
+  <div class="field">
+    <%= f.label :first_name %><br />
+    <%= f.text_field :first_name, autofocus: true, autocomplete: "first name" %>
```

Comment:
use placeholders in input fields

Reply:
This was generated by devise and will be modified after merge.